Fakebook.Profile/Fakebook.Profile.Domain/IRepository.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Threading.Tasks;
+
+namespace Fakebook.Profile.Domain
+{
+    /// <summary>
+    /// Interface for data retrieval repository.
+    /// </summary>
+    public interface IRepository
```

Comment:
name should be more specific, since there's data that doesn't come through this repository (images).